packages/integration-tests/suites/tracing/browsertracing/navigation/test.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { getSentryTransactionRequest } from '../../../../utils/helpers';
+
+sentryTest('should create a navigation transaction on page navigation', async ({ getLocalTestPath, page }) => {
+  const url = await getLocalTestPath({ testDir: __dirname });
+
+  // This is the `pageload` transaction
+  await getSentryTransactionRequest(page, url);
+
+  const navigationRequest = await getSentryTransactionRequest(page, `${url}#foo`);
```

Comment:
I'd rather us use the multiple transactions helper here.

I would also like to see some more assertions to show the differences between pageload and navigation transactions.

Reply:
> I'd rather us use the multiple transactions helper here.

My intention here is to initiate a navigation (to `#foo`) after the `pageload` transaction, so it's not possible to use the helper as it operates on a single URL on all requests.

Reply:
Ah got it - my mistake.